hazelcast/src/test/java/com/hazelcast/test/starter/HazelcastVersionLocator.java

```diff
@@ -79,41 +80,24 @@ private static File locateMember(String version, File target, boolean enterprise
     }
 
     // attempts to locate member tests artifact in local maven repository, then downloads
-    private static File locateMemberTests(String version, File target, boolean enterprise) {
-        File artifact = new File(LOCAL_M2_REPOSITORY_PREFIX + constructPathForMemberTests(version, enterprise));
+    private static File locateTestsArtifact(String version, File target, boolean enterprise) {
```

Comment:
Minor: since `locateMemberTests` has been renamed to `locateTestsArtifact`, maybe `downloadMemberTests` should be renamed, too?